extensions/mssql/yarn.lock

```diff
@@ -189,10 +189,10 @@
   dependencies:
     "@vscode/extension-telemetry" "0.6.1"
 
-"@microsoft/ads-service-downloader@^1.1.0":
-  version "1.1.0"
-  resolved "https://registry.yarnpkg.com/@microsoft/ads-service-downloader/-/ads-service-downloader-1.1.0.tgz#c51ba28bc17a137e92f99a2b050821965588d7d6"
-  integrity sha512-WSJ4NZvpuebg/CjlTUpqXPUYV5y5mSNJQE/dd+kuOt0nkMYzCmp5IYyFFpMFUR1CiZy/19V7DH2wGdy60+9SRg==
+"@microsoft/ads-service-downloader@^1.2.0":
```

Comment:
I'd suggest also updating all the other extensions that use the service downloader as well, keeping everything in sync is typically a good idea to ensure consistent behavior.

Reply:
sounds reasonable. I've updated other extensions using this package and updated the config file so that they will work just like before.